include/robot_interfaces/n_joint_robot_types.hpp

```diff
@@ -19,6 +19,33 @@
 
 #include <robot_interfaces/robot_logger.hpp>
 
+// https://stackoverflow.com/a/51944389/2095383
+// Authors: Azoth, eudoxos
+// Date: 2020-01-15
+// License: CC BY-SA 4.0
+namespace cereal
```

Comment:
I just noticed that these serialization functions are still here. They should be moved to a more central place as it might be useful in other places.  I'll do this and update here.